src/format.js

```diff
@@ -72,9 +73,8 @@ export function formatTokenAmount(
         )
       )
 
-  const leftPart = formatNumber(
-    JSBI.divide(amountConverted, JSBI.exponentiate(_10, digits))
-  )
+  const leftPart = JSBI.divide(amountConverted, JSBI.exponentiate(_10, digits))
+  const processedLeftPart = commify ? formatNumber(leftPart) : leftPart
```

Comment:
Let's add tests for this new commify option.

Reply:
Done in https://github.com/aragon/token-amount/pull/7/commits/545a6e848ca3e90a21bf0012edd5785c571fb193